src/utilities/utilities.c

```diff
@@ -169,6 +188,9 @@ hypre_CreateNextDirOfSequence(const char *basepath, const char *prefix, char **f
       }
    }
    closedir(dir);
+#else
+   /* TODO (VPM) */
```

Comment:
@rfalgout I will implement this code block in another PR. The intention here is mainly to fix compilation in Windows. Sorry for overlooking this before...